src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -161,6 +285,14 @@ public static MythicMob getMythicMob(String name) {
         return MythicBukkit.inst().getMobManager().getMythicMob(name).orElse(null);
     }
 
+    public static Map<String, Double> getDamageModifiers(MythicMob mob) {
+        return mob.getDamageModifiers();
```

Comment:
Is there a reason to have this method instead of calling `MythicMob#getDamageModifiers` directly?